schema/src/db/models.rs.orig

```diff
@@ -0,0 +1,286 @@
+            .values(self)
+            .execute(conn)?;
+        Ok(result)
+    }
+
+    pub fn sql_fragment(&self) -> String {
+        if self.nullable {
+            format!("{} {}", self.column_name, self.sql_type())
+        } else {
+            format!("{} {} not null", self.column_name, self.sql_type())
+        }
+    }
+
+    fn sql_type(&self) -> &str {
+        match self.column_type {
+            ColumnType::ID => "bigint primary key",
+            ColumnType::Address => "varchar(64)",
+            ColumnType::Bytes4 => "varchar(8)",
+            ColumnType::Bytes8 => "varchar(16)",
+            ColumnType::Bytes32 => "varchar(64)",
+            ColumnType::AssetId => "varchar(64)",
+            ColumnType::ContractId => "varchar(64)",
+            ColumnType::Salt => "varchar(64)",
+            ColumnType::Int4 => "integer",
+            ColumnType::Int8 => "bigint",
+            ColumnType::UInt4 => "integer",
+            ColumnType::UInt8 => "bigint",
+            ColumnType::Timestamp => "timestamp",
+            ColumnType::Blob => "bytea",
+        }
+    }
+}
+
+#[derive(Queryable, QueryableByName)]
+#[table_name = "graph_registry_columns"]
+#[allow(unused)]
+pub struct Columns {
+    pub id: i32,
+    pub type_id: i64,
+    pub column_position: i32,
+    pub column_name: String,
+    pub column_type: ColumnType,
+    pub nullable: bool,
+    pub graphql_type: String,
+}
+
+impl Columns {
+    pub fn list_by_id(conn: &PgConnection, col_id: i64) -> QueryResult<Vec<Columns>> {
+        use gr::graph_registry_columns::dsl::*;
+        graph_registry_columns.filter(type_id.eq(col_id)).load(conn)
+    }
+}
+
+#[derive(Debug, Queryable, QueryableByName)]
+pub struct ColumnInfo {
+    #[sql_type = "BigInt"]
+    pub type_id: i64,
+    #[sql_type = "Text"]
+    pub table_name: String,
+    #[sql_type = "Integer"]
+    pub column_position: i32,
+    #[sql_type = "Text"]
+    pub column_name: String,
+    #[sql_type = "Columntypename"]
+    pub column_type: ColumnType,
+}
+
+impl ColumnInfo {
+    pub fn get_schema(
+        conn: &PgConnection,
+        name: &str,
+        version: &str,
+    ) -> QueryResult<Vec<ColumnInfo>> {
+        use gr::graph_registry_columns::dsl as cd;
+        use gr::graph_registry_type_ids::dsl as td;
+
+        let result = td::graph_registry_type_ids
+            .inner_join(cd::graph_registry_columns.on(cd::type_id.eq(td::id)))
+            .select((
+                cd::type_id,
+                td::table_name,
+                cd::column_position,
+                cd::column_name,
+                cd::column_type,
+            ))
+            .filter(td::schema_name.eq(name).and(td::schema_version.eq(version)))
+            .order((cd::type_id, cd::column_position))
+            .load::<_>(conn)?;
+
+        Ok(result)
+    }
+}
+
+#[derive(Debug, QueryableByName)]
+pub struct EntityData {
+    #[sql_type = "Binary"]
+    pub object: Vec<u8>,
+}
```

Comment:
Is there any reason why we still need to keep this file? seems to have git conflicts baked into the file as well...